### ISSUE TITLE
g++ 's error

### ISSUE THREAD

**Issue by 9p15p** [MinghanLi/STMask#3]
I found a problem.
This code runs uncorrectly in `Pytorch1.5.0`
please use `Pytorch 1.4.0`
and when install mmcv, please use `pip install mmcv-full==1.1.2 -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.4.0/index.html`

**Comment by zhaiyukun** [comment]
have you run the train code sucessfully? @9p15p

**Comment by MinghanLi** [comment]
If you want to use Pytorch with higher versions, please mask sure that the corresponding version of mmcv-full, DCN and spatial-correlation-sampler packages are installed! All related links are listed below for your convenience. BTW, the code can run in Pytorch1.8.0.

[https://github.com/open-mmlab/mmcv](url)
[https://github.com/jinfagang/DCNv2_latest](url)
[https://github.com/ClementPinard/Pytorch-Correlation-extension](url)
